pkg/helper/client.go

```diff
@@ -112,7 +112,7 @@ func set(desiredState nmstatev1alpha1.State) (string, error) {
 		// commit timeout doubles the default gw ping probe timeout, to
 		// ensure the Checkpoint is alive before rolling it back
 		// https://nmstate.github.io/cli_guide#manual-transaction-control
-		output, err = nmstatectl([]string{"set", "--no-commit", "--timeout", strconv.Itoa(defaultGwProbeTimeout * 2)}, string(desiredState.Raw))
+		output, err = nmstatectl([]string{"set", "--no-commit", "--timeout", strconv.Itoa((defaultGwProbeTimeout / time.Second) * 2)}, string(desiredState.Raw))
```

Comment:
No need for this monkey business https://golang.org/pkg/time/#Duration.Seconds

Reply:
Done